### PR TITLE
i18n: modify/add literals from locdiff

### PR DESCRIPTION
**Pull request by rohank07**
### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

Update/add UI literals changed from Juliet Rose initial release (2022.02.0+443) using [locdiff](https://github.com/rstudio/rstudio/blob/0c04daa598130b037229168fd32e29f1d2900a71/src/gwt/tools/locdiff/locdiff.js) script . 

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests

cc @gtritchie

### CODE REVIEW

**Comment by rohank07** [comment]
In src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessorConstants_en.properties the UI literals that have single quotes are escaped not with `'`. Literals that consits of a single quote in the English and French must be escaped. I modified src/cpp/session/resources/schema/user-prefs-schema.json to correct these, however I'd need someone to run `generate-prefs.R` (my build is not functional atm) to rebuild the property file UserPrefsAccessorConstants_en.

**Comment by gtritchie** [comment]
> however I'd need someone to run `generate-prefs.R` (my build is not functional atm) to rebuild the property file UserPrefsAccessorConstants_en.

Just did that and pushed it to your branch.

File: src/cpp/session/resources/schema/user-prefs-schema.json [hunk 1]
```diff
@@ -1596,10 +1596,10 @@
             "enum": ["en", "fr"],
             "default": "en",
             "title": "User Interface Language:",
-            "description": "The IDE's user-interface language."
+            "description": "The IDE''s user-interface language."
         },
         "native_file_dialogs": {
-            "description": "Whether RStudio Desktop will use the operating system's native File and Message dialog boxes.",
+            "description": "Whether RStudio Desktop will use the operating system''s native File and Message dialog boxes.",
```
Comment (gtritchie) [inline src/cpp/session/resources/schema/user-prefs-schema.json:1602]: Putting double-apostrophes in the user-prefs-schema.json is causing double-aprostrophes to show up in the UI. For example, I built this branch and see this in the tooltip:

Screen Shot 2022-05-31 at 4 55 02 PM

**Review by gtritchie — CHANGES_REQUESTED** [review]
The double apostrophes in the user-prefs-schema is showing up that way in the UI (as double apostrophes). So I don't think we want to do that.

File: src/cpp/session/resources/schema/user-prefs-schema.json (hunk 1 above)
Comment (rohank07) [reply src/cpp/session/resources/schema/user-prefs-schema.json:1602]: ahh I see. I will have this reverted in a bit!

**Review by romainfrancois — CHANGES_REQUESTED** [review]
cc @cderv

File: src/gwt/src/org/rstudio/studio/client/workbench/views/jobs/JobsConstants_fr.properties [hunk 2]
```diff
@@ -2,7 +2,7 @@ localJobsRunningCaption=Travaux locaux encore en cours d''exécution
 localJobsRunningMessage=L''onglet Travaux ne peut pas être fermé tant qu''il {0}\n\nAttendez que tous les travaux locaux soient terminés.
 localJobsUnfinished=Ce sont des travaux locaux non terminés.
 localJobUnfinished=est un travail local inachevé.
-jobsTitle=Travaux
+jobsTitle=Emplois locaux
```
[the remark this answers is not in the document]
Comment (cderv) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/jobs/JobsConstants_fr.properties:5]: I would also use "Tâches" and not "Travaux" -> `Tâches locales`

**Review by gtritchie — APPROVED** [review]

**Pull request merged**